conf/battle/feature.conf

```diff
@@ -67,6 +67,10 @@ feature.roulette: on
 // Requires: 2015-05-13aRagexe or later
 feature.achievement: on
 
+// Equipment Switch (Note 1)
+// Requires: 2017-02-08bRagexeRE or later
+feature.equipswitch: off
```

Comment:
The feature is not available on official server by default?

Reply:
I think this is because of the warning that’s presented at start up since our packetver is 2015~ and it’s causing the build bots to “fail”.